### PR TITLE
Update content in notification banner

### PR DESCRIPTION
**Pull request by samuelhwilliams**
## Summary
Update copy in the notification banner on service/lots page of supplier application to framework to include links to the company details page and the declaration page

## Example
screen shot 2018-03-01 at 11 50 14

## Associated PR
~https://github.com/alphagov/digitalmarketplace-frontend-toolkit/pull/412~

## Ticket
https://trello.com/c/JVr9gFEC/56-incomplete-application-banner

### CODE REVIEW

File: tests/app/main/test_frameworks.py [hunk 1]
```diff
@@ -3776,6 +3776,38 @@ def test_should_be_a_503_if_email_fails(self, send_email, data_api_client):
 @mock.patch('app.main.views.frameworks.data_api_client', autospec=True)
 @mock.patch('app.main.views.frameworks.count_unanswered_questions')
 class TestG7ServicesList(BaseApplicationTest):
+    def _assert_incomplete_application_banner_not_visible(self, doc):
+        assert not doc.xpath('//*[@class="banner-information-without-action"]')
```
Comment (TheDoubleK) [inline tests/app/main/test_frameworks.py]: "Xpath doesn't exist" is quite a brittle test, easy to get false positives (e.g. the mark up changes, the banner shows up when it shouldn't, but this tasty still says everything is fine). Might be good to have a slightly "blunter" check for the banner heading text anywhere in the page.

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "colors": "1.1.2",
     "jquery": "1.12.0",
     "hogan.js": "3.0.2",
-    "digitalmarketplace-frontend-toolkit": "https://github.com/alphagov/digitalmarketplace-frontend-toolkit.git#v28.3.0",
+    "digitalmarketplace-frontend-toolkit": "https://github.com/alphagov/digitalmarketplace-frontend-toolkit.git#v28.5.1",
```
[the remark this answers is not in the document]
Comment (samuelhwilliams) [reply package.json]: Yep

File: tests/app/main/test_frameworks.py (hunk 1 above)
Comment (samuelhwilliams) [reply tests/app/main/test_frameworks.py]: Fair point, will update

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (samuelhwilliams) [reply package.json]: Done.

**Comment by samuelhwilliams** [comment]
Content confirmed by @constancecerf

File: tests/app/main/test_frameworks.py [hunk 3]
```diff
@@ -3776,6 +3776,38 @@ def test_should_be_a_503_if_email_fails(self, send_email, data_api_client):
 @mock.patch('app.main.views.frameworks.data_api_client', autospec=True)
 @mock.patch('app.main.views.frameworks.count_unanswered_questions')
 class TestG7ServicesList(BaseApplicationTest):
+    def _assert_incomplete_application_banner_not_visible(self, html):
+        assert "Your application is not complete" not in html
```
Comment (TheDoubleK) [inline tests/app/main/test_frameworks.py]: Would also be nice to have the exact opposite assertion, ie `assert "Your application is not complete" in html`, in the tests for when banner is visible.

Comment (samuelhwilliams) [reply tests/app/main/test_frameworks.py]: Done.

**Pull request merged**